.travis.yml

```diff
@@ -5,6 +5,9 @@ php:
   - 7.2
   - 7.3
   - 7.4
+  - 8.0
+  - 8.1
+  - 8.2
```

Comment:
Travis CI is no longer used, but I'll replace this with a GitHub Action config in a subsequent commit.

Reply:
Fixed in #22.